ui/src/views/compute/StartVirtualMachine.vue

```diff
@@ -16,7 +16,7 @@
 // under the License.
 
 <template>
-  <div class="form-layout" v-ctrl-enter="handleSubmit">
```

Comment:
@DaanHoogland from my tests, `v-ctrl+enter` only works if something in the component is focused. I'm adding focus to the `Consider Last Host` switch (just for convenience, as it’s unrelated to the fix), but this alone is not enough to use `v-ctrl+enter` because the User role doesn’t have access to this option. Actually, the `Start Instance` dialog for Users only has the `Ok` and `Cancel` buttons.

I considered setting focus on one of these buttons, but that could give room for mistakes, specially the `Ok`. Futhermore, if the `Ok` button is focused, pressing `Enter` would be enough, making `v-ctrl-enter` unnecessary.